### ISSUE TITLE
Cannot scroll to bottom of tag information

### ISSUE THREAD

**Issue by jidanni** [MarcusWolschon/osmeditor4android#2259]
![Screenshot_20230429-101523.jpg](https://user-images.githubusercontent.com/1230959/235279088-b025ab8f-51d8-4be9-93de-cfe2caf1adc7.jpg)

https://openstreetmap.org/node/8535431426

**Comment by jidanni** [comment]
But here I can,
![Screenshot_20230429-102000.jpg](https://user-images.githubusercontent.com/1230959/235279261-486d2809-4fd6-4a1d-a32c-cd4b569b36bf.jpg)

So the bug only occurs if there is exactly the triggering amount of lines.

**Comment by jidanni** [comment]
As you can see I'm not using any unreasonably large size fonts or anything. So don't ask me about those :-)

**Comment by simonpoole** [comment]
"The program" does nothing with both font size and magnification (I need to turn both on to make this happen), it simply uses standard Android libraries here which have a bug that will slightly cut off the last line.

**Comment by jidanni** [comment]
Should I report it to them?
Has anybody reported it?

**Comment by simonpoole** [comment]
I haven't (and to find out if someone else has you are might have to spend a couple of hours searching through googles issue tracker).

If you ask me there's no point, but if you want to do so you will need to write a simple test case (aka an app) that demonstrates the problem for anything at all to happen.

**Comment by jidanni** [comment]
Well I found https://issuetracker.google.com/issues/236645847 , maybe related.
But there may be faster ways to Workaround the Android Bug.
https://stackoverflow.com/questions/32631788/android-textview-text-cut-off-at-bottom
https://github.com/facebook/react-native/issues/17064
https://teamtreehouse.com/community/when-you-see-cutoff-at-the-bottom-of-your-textview
https://www.google.com/search?q=android+text+cut+off etc.

**Comment by jidanni** [comment]
Perhaps a blank line could be added as a temporary workaround until the real source of the Android bug is found.
It is better than the user needing to add some "zzz=nevermind" tag in order to shove the rest up to be fully visible.

**Comment by simonpoole** [comment]
This doesn't seem to happen with the current version. @jidanni can you check?

**Comment by jidanni** [comment]
Yes I will check soon. Can I please ask you a personal request, can you please stop blocking me. It makes very hard to access things like this notification.
![Screenshot_20240108_082703_Settings.jpg](https://github.com/MarcusWolschon/osmeditor4android/assets/1230959/6663f6e1-ae3f-4cc2-b043-0b871c21badc)

You see the usual notifications when people comment on things, I don't get them anymore. And then the only way I knew that you asked me this question is because I got it in my cell phone notifications from the GitHub app. But then the second time I wanted to look at it, it just brought me to the app without going to the item .

**Comment by jidanni** [comment]
As far as the original problem goes, here's what it looks like now,
![Screenshot_20240108_143124_Vespucci.jpg](https://github.com/MarcusWolschon/osmeditor4android/assets/1230959/f6c0c661-b7eb-4609-a0f8-c9bd49bd7c16)
But that's not long enough to tell if the bug is still there.
But yes please close this bug for now. But do read what I posted above about the GitHub blocking bug. Thanks.
